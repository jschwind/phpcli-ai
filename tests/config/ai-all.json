{
  "exclude": {
    "extensions": [],
    "folders": [],
    "filenames": ["/lvl1/lvl2/test.php"]
  },
  "include": {
    "extensions": ["php"],
    "folders": [],
    "filenames": []
  }
}