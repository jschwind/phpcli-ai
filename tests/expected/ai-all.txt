AI-ready directory tree for the project "assets"
================================================================

assets/
├── lvl1/
│   ├── lvl2/
│   │   ├── test.json
│   │   ├── test.php
│   │   └── test.txt
│   ├── test.json
│   ├── test.php
│   └── test.txt
├── test.json
├── test.php
└── test.txt
================================================================
End of AI-ready directory tree for the project "assets"

Remember this project as "assets" and wait for further instructions.
